scripts/bookmarklet.tmpl.js

```diff
@@ -0,0 +1,54 @@
+/* global jQuery, window, document */
+(function(window, document) {
+  var toast = document.createElement('div');
+  var b = document.getElementsByTagName('body')[0];
+  var version = null;
+
+  setStyles(toast)([
+    ['position' , 'fixed'],
+    ['top' , '20px'],
+    ['right' , '30%'],
+    ['padding' , '5px 10px'],
+    ['zIndex' , 1001],
+    ['fontSize' , '18px'],
+    ['color' , '#222'],
+    ['backgroundColor' , '#DFF2BF'],
+    ['text-align', 'center'],
+    ['border', 'solid 2px black'],
+    ['border-radius', '25px']
+  ]);
```

Comment:
Since `setStyles` is only used in one place we could simply set these style properties directly:

```javascript
toast.style.position = 'fixed';
toast.style.top = '20px';
toast.style.right = '30%';
...
```

Also, why `right: 30%`? I'd have expected the notification to appear in the top-right corner.